helm-service/controller/onboarder.go

```diff
@@ -88,10 +89,26 @@ func (o *Onboarder) DoOnboard(ce cloudevents.Event, loggingDone chan bool) error
 		}
 	}
 
+	kubeClient, err := keptnutils.GetKubeAPI(true)
+	if err != nil {
+		return err
+	}
 	for _, stage := range stages {
 		if err := o.onboardService(stage.StageName, event, url.String()); err != nil {
 			return err
 		}
+		if o.isBlueGreenStage(event.Project, stage.StageName) {
```

Comment:
Please add a comment here, e.g.: ``// inject istio to the namespace for blue-green deployments``